Example/PaymentSheet Example/PaymentSheet Example/ExampleCustomDeferredCheckoutViewController.swift

```diff
@@ -0,0 +1,334 @@
+                    let subtotal = json["subtotal"] as? Double,
+                    let tax = json["tax"] as? Double,
+                    let total = json["total"] as? Double,
+                    let self = self
+                else {
+                    // Handle error
+                    return
+                }
+
+                self.computedTotals = ComputedTotals(subtotal: subtotal, tax: tax, total: total)
+
+                // MARK: Set your Stripe publishable key - this allows the SDK to make requests to Stripe for your account
+                STPAPIClient.shared.publishableKey = publishableKey
+
+                // MARK: Create a PaymentSheet.FlowController instance
+                var configuration = PaymentSheet.Configuration()
+                configuration.merchantDisplayName = "Example, Inc."
+                configuration.applePay = .init(
+                    merchantId: "com.foo.example", merchantCountryCode: "US")
+                configuration.customer = .init(
+                    id: customerId, ephemeralKeySecret: customerEphemeralKeySecret)
+                configuration.returnURL = "payments-example://stripe-redirect"
+                // Set allowsDelayedPaymentMethods to true if your business can handle payment methods that complete payment after a delay, like SEPA Debit and Sofort.
+                configuration.allowsDelayedPaymentMethods = true
+                DispatchQueue.main.async {
+                    PaymentSheet.FlowController.create(
+                        intentConfig: self.intentConfig,
+                        configuration: configuration
+                    ) { [weak self] result in
+                        switch result {
+                        case .failure(let error):
+                            print(error)
+                        case .success(let paymentSheetFlowController):
+                            self?.paymentSheetFlowController = paymentSheetFlowController
+                            self?.paymentMethodButton.isEnabled = true
+                            self?.hotDogStepper.isEnabled = true
+                            self?.saladStepper.isEnabled = true
+                            self?.subscribeSwitch.isEnabled = true
+                            self?.updateButtons()
+                        }
+                    }
+                }
+            })
+
+        task.resume()
+    }
+
+    func createIntent(paymentMethodID: String, completion: @escaping (Result<String, Error>) -> Void) {
+        var request = URLRequest(url: createIntentUrl)
+        request.httpMethod = "POST"
+
+        var body: [String: Any?] = [
+            "payment_method_id": paymentMethodID,
+            "currency": "USD",
+            "hot_dog_count": hotDogStepper.value,
+            "salad_count": saladStepper.value,
+            "is_subscribing": subscribeSwitch.isOn,
+        ]
+
+        if subscribeSwitch.isOn {
+            body["setup_future_usage"] = "off_session"
+        }
```

Comment:
do we need this? isn't it already handled by is_subscribing?

Reply:
Yep, after moving the amount calculation to the backend I didn't make this connection.